maqs-utilities/src/main/java/com/cognizantsoftvision/maqs/utilities/logging/FileLogger.java

```diff
@@ -241,35 +242,45 @@ public FileLogger(boolean append, String logFolder, String name, MessageType mes
   }
 
   /**
-   * {@inheritDoc}
```

Comment:
You want to inherit docs in these cases - That way we don't need to maintain them on every interface

Reply:
added back inheritDoc comments